truss/truss_config.py

```diff
@@ -156,6 +156,7 @@ class ModelServer(Enum):
     TGI = "TGI"
     VLLM = "VLLM"
     TRITON = "TRITON"
+    TRT_LLM = "TRT_LLM"
```

Comment:
Worth adding a comment [here](https://github.com/basetenlabs/truss/blob/189df18fe2a7f88479ddaf886af0a61c26285e9b/truss/contexts/image_builder/serving_image_builder.py#L500) that the build for `TrussServer` and `TRT_LLM` use the default buildir setup.

I wish [this line](https://github.com/basetenlabs/truss/blob/189df18fe2a7f88479ddaf886af0a61c26285e9b/truss/remote/baseten/remote.py#L56) showed up in this diff somewhere so it's clear why this works.